settings.php

```diff
@@ -41,6 +41,12 @@
         '')
     );
 
+    $settings->add(new admin_setting_configcheckbox('local_login/forcelogin',
+        get_string('forcelogin', 'local_login'),
+        get_string('forcelogin_desc', 'local_login'),
+        1)
```

Comment:
can we change the default to 0 here - not many people will want this particular setting (I think) - most people will likely use the main Moodle forcelogin setting which forces login for "all pages" not just the homepage like we're doing here.